expat/coverage.sh

```diff
@@ -110,8 +110,8 @@ _run() {
         cd "${build_dir}"
 
         _configure \
-                CFLAGS="${BASE_FLAGS}" \
-                CXXFLAGS="${BASE_FLAGS}"
+                CFLAGS="${CFLAGS}" \
+                CXXFLAGS="${CXXFLAGS}"
```

Comment:
I believe use of `BASE_*` was intended here, to keep `-std=*` out of configure tests. I'll drop that patch. Happy to re-add if I'm missing something here.

Reply:
Just found our discussion at https://github.com/libexpat/libexpat/pull/229/files/4dc2104a146f9e7902b73c54749f46abaf15a5ff#diff-023efc1ccece8957b2bd273ffe381a61 now and applied your patch as well at 3f8e28f3cbcd29cecfc425886efac95d63c57bb6.